buildtools/webpack.ngeoexamples.js

```diff
@@ -2,23 +2,44 @@ const path = require('path');
 const ls = require('ls');
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 const plugins = [];
 const entry = {};
 
 for (const filename of ls('examples/*.html')) {
   const name = filename.name;
-  entry[name] = ['ngeo/mainmodule.js', `./examples/${name}.js`];
+  entry[name] = [
+    './examples/common_dependencies.js',
+    'ngeo/mainmodule.js', // TODO check path!
+    `./examples/${name}.js`
+  ];
   plugins.push(
     new HtmlWebpackPlugin({
       template: `examples/${name}.html`,
       chunksSortMode: 'manual',
       filename: name + '.html',
       chunks: ['common', name],
-    }),
+    })
   );
 }
 
+// move data folder
+plugins.push(new CopyWebpackPlugin(
+  [
+    {
+      from: 'examples/data/*',
+      to: 'data',
+      flatten: true
+    },
+    {
+      from: 'examples/partials/*',
```

Comment:
And this one is relay needed?
The partials are loaded throw requires...
It's not working?

Reply:
I tested it again, and for the partials inside the examples, this is needed. Those HTML files were loaded during runtime and not included by webpack.

Reply:
Looks strange, but I cans see it later...